session/session.go

```diff
@@ -112,16 +112,16 @@ func New(
 	}
 }
 
-// SetAvailableHTTPSBouncer sets the HTTP bouncer base URL.
-func (s *Session) SetAvailableHTTPSBouncer(baseURL string) {
+// AddAvailableHTTPSBouncer adds the HTTP bouncer base URL.
```

Comment:
@xhdix I just noticed that this documentation comment does not make much sense now. Can you make it a bit more specific saying that the URL is added to the list of URLs that are tried?

Reply:
@bassosimone sure! done.